src/components/MySpace/MySpace.tsx

```diff
@@ -86,6 +86,17 @@ const MySpace = ({ bookmarks }: { bookmarks: CMSBookmark[] }) => {
     <div id="skip-announcements-carousel" className={styles.mySpace}>
       <div className={styles.widgetContainer}>
         <h2 className={styles.pageTitle}>My Space</h2>
+
```

Comment:
This is for demo/testing purposes to show @jcbcapps the addNewWeatherWidget functionality. Will remove once we've had a walk-through and he's branched.